integrations/openaiwrapper.py

```diff
@@ -20,53 +22,67 @@ def __init__(self, api_key, timeout=10):
         self.api_key = api_key
         openai.api_key = api_key
         self.timeout = timeout
-        self.cache = {}
 
+    @memoize_to_sqlite(func_name="get_embedding", filename="openai_embedding_cache.db")
     def get_embedding(self, text):
         """
         Retrieves the embedding for the given text.
 
         :param text: The text for which embedding is required.
         :return: The embedding for the given text.
         """
-        if text in self.cache:
-            return self.cache[text]
-
         start_time = time.time()
         retries = 0
 
         while time.time() - start_time < self.timeout:
             try:
-                embedding = openai.Embedding.create(input=text, engine="text-embedding-ada-002")
-                self.cache[text] = embedding
-                return embedding
+                return openai.Embedding.create(input=text, engine="text-embedding-ada-002")
             except openai.error.OpenAIError as e:
                 logging.error(f"OpenAI API error: {e}")
                 retries += 1
                 if retries >= MAX_RETRIES:
                     raise
                 time.sleep(RETRY_SLEEP_DURATION)
-        
+                #for those using the free version of openai
```

Comment:
Not sure what to do here.  Do you want me to remove the commented code or uncomment the commented code so that it executes if a Rate limit occurs?   For now I will uncomment the code and can remove it later if that is what you want.